### ISSUE TITLE
Brilliant flash

### ISSUE THREAD

**Issue by theemack** [maxgrafik/violetCMS#2]
I'm eager to try out your cms, followed your instructions as simple as they are but upon login in my anticipation of the default (admin) to see this cms in action...I'm presented with a brilliant flash of the screen only to default back to the login panel with no error codes like an asteroid I am combing the files to see what I might pick up...alas. Are there any other fixes i need to correct that are not stated in your write up...?

**Comment by maxgrafik** [comment]
Hi, I assume the front page is working, but only login to the backend doesn't work. Sounds like an issue with authentication, if you see a short flash of the admin backend and are immediately sent back to the login page. May I ask which browser you are using? Make sure to allow cookies, as they are required for authentication. Also, make sure the .htaccess file in the `violet` sub directory is present, so the authorization headers are recognized by the backend. This is what first comes to my mind. Let me know if this already helped. I may have to dig deeper ;)

**Comment by theemack** [comment]
Thanks for your prompt response, currently running he latest Firefox browser on Fedora36, installed a fresh chromium too - the result was the same flash, last attempt was to login on TOR Browser - the result was exactly the same as well but allowed me to see the dashboard for a mere second defaulting back to the login screen. All files in the rightful directory and alternated different php versions to eliminate tangents...

**Comment by maxgrafik** [comment]
Alright, there's obviously a serious bug somewhere. It seems you can log in, but the very first request after that (dashboard) is rejected and thus you're sent back to login screen. To be honest I'm a bit clueless about where in the process the request fails exactly.
I just pushed a new debug branch. The file that changed is `violet/system/Ajax/APIAccess.php` which should create a `debug.log` file in `violet/logs`. This won't solve the problem, but at least it logs where the request fails. Could you please try replace the file and report back? I'm sorry for the inconvenience.

**Comment by theemack** [comment]
[REDACTED]

Note: @maxgrafik edited this comment

**Comment by maxgrafik** [comment]
OK sorry, my instruction was probably unclear. What I meant was:
- Download [this version](https://github.com/maxgrafik/violetCMS/archive/refs/heads/debug.zip) please
- Replace your current installation of violetCMS with the newly downloaded files
- Try to login (which *will* fail)
- Then look in `/violet/logs` for a new file named `debug.log` and post its contents

**Comment by theemack** [comment]
Great - Managed to log in and had a look around, 

- A few issues tho in that it automatically logs you out should you open another tab to view something else within 30secs or so.

- The preview button does not allow for edited previews of the page
- On logout pages created show up as 500 errors
- Only the home page is the visible edited page
- Page links created are visible in the menu but do not exist
- Fails to resize to mobile screen allowing for horizontal scroll only.
- /violet/logs below:
- 217.160.152.38 - - [06/Sep/2022:10:56:37 +0200] "GET / HTTP/1.1" 200 2175 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.75.14 (KHTML, like Gecko) Version/7.0.3 Safari/7046A194A"
217.160.152.38 - - [06/Sep/2022:10:56:37 +0200] "GET / HTTP/1.1" 200 2175 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.75.14 (KHTML, like Gecko) Version/7.0.3 Safari/7046A194A"
2a00:23c7:e508:8e01:33ec:a5a3:98c1:1401 - - [06/Sep/2022:10:56:46 +0200] "GET / HTTP/1.1" 200 2175 "https://myhost/" "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"

**Comment by maxgrafik** [comment]
Great. Thanks for the feedback.
* I added a `X-Frame-Options` reset to `.htaccess`. Preview should be fixed now.
* It's (currently) not possible to be logged in on multiple tabs at the same time, yet.
* Mobile screen resolutions work for me (both front page and admin backend). Tested on iPhone, but also on Safari/Chrome/Firefox in dev mode

Can't reproduce the other problems, though. Sure you have set the pages as `visible` and `published` under `options`?
